5.5/Dockerfile

```diff
@@ -32,20 +29,23 @@ RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/mysql55/epel-7-x86_64/download/rhscl-mysql55-epel-7-x86_64.noarch.rpm && \
     yum -y --setopt=tsflags=nodocs install gettext hostname bind-utils mysql55 && \
     yum clean all && \
-    mkdir -p /var/lib/mysql/data && chown mysql.mysql /var/lib/mysql/data && \
-    test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)" && \
-    chmod o+w -R /var/lib/mysql
+    test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)"
 
 # When bash is started non-interactively, to run a shell script, for example it
 # looks for this variable and source the content of this file. This will enable
 # the SCL for all scripts without need to do 'scl enable'.
-ENV BASH_ENV=/var/lib/mysql/scl_enable \
-    ENV=/var/lib/mysql/scl_enable \
-    PROMPT_COMMAND=". /var/lib/mysql/scl_enable"
+ENV BASH_ENV=/usr/share/cont-lib/cont-env.sh \
+    ENV=/usr/share/cont-lib/cont-env.sh \
+    PROMPT_COMMAND=". /usr/share/cont-lib/cont-env.sh" \
+    MYSQL_SCL=mysql55
+
+ADD root /
+
+RUN /usr/libexec/cont-setup
```

Comment:
move the ADD above the first RUN and then merge this RUN into the existing RUN?  Saves a layer which isn't critical since we squash, but it makes the build faster too.

(You'll have to move the ENV up too since cont-setup depends on it, i don't think that has any negative ramifications though)